tools/gatk4.wdl

```diff
@@ -0,0 +1,358 @@
+## [Homepage](https://software.broadinstitute.org/gatk)
+#
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
+version 1.1
+
+task split_n_cigar_reads {
+    meta {
+        description: "Splits reads that contain Ns in their CIGAR strings into multiple reads."
+        external_help: "https://gatk.broadinstitute.org/hc/en-us/articles/360036858811-SplitNCigarReads"
+        outputs: {
+            split_n_reads_bam: "BAM file with reads split at N CIGAR elements and updated CIGAR strings."
+            split_n_reads_bam_index: "Index file for the split BAM"
+            split_n_reads_bam_md5: "MD5 checksum for the split BAM"
+        }
+    }
+
+    parameter_meta  {
+        bam: "Input BAM format file to with unsplit reads containg Ns in their CIGAR strings."
+        bam_index: "BAM index file corresponding to the input BAM"
+        fasta: "Reference genome in FASTA format. Must be uncompressed."
+        fasta_index: "Index for FASTA format genome"
+        dict: "Dictionary file for FASTA format genome"
+        interval_list: "Interval list indicating regions in which to split reads"
+        prefix: "Prefix for the BAM file. The extension `.bam` will be added."
+        modify_memory_gb: "Add to or subtract from dynamic memory allocation. Default memory is determined by the size of the inputs. Specified in GB."
+        ncpu: "Number of cores to allocate for task"
+    }
+
+    input {
+        File bam
+        File bam_index
+        File interval_list
+        File fasta
+        File fasta_index
+        File dict
+        String prefix = basename(bam, ".bam") + ".split"
+        Int modify_disk_size_gb = 0
+        Int ncpu = 8
+    }
+
+    Int disk_size_gb = ceil(size(bam, "GB") + 1) * 3 + ceil(size(fasta, "GB")) + modify_disk_size_gb
+
+    command <<<
+        set -euo pipefail
+
+        gatk \
```

Comment:
Is there a reason this command doesn't have `--java-options` like the other `gatk` commands?

Reply:
Not a good one. I've added it.